proposals/description/pod_exec_v2.ts

```diff
@@ -0,0 +1,47 @@
+import { TemplatedProposalDescription } from '@custom-types/types';
+
+const pod_executor_v2: TemplatedProposalDescription = {
+  title: 'Pod Executor V2',
+  commands: [
+    {
+      target: 'tribalCouncilTimelock',
+      values: '0',
+      method: 'grantRole(bytes32,address)',
+      arguments: (addresses) => [
```

Comment:
Let's start using `ethers.utils.id` for roles instead of the raw bytes

Reply:
Good idea, will do going forward